openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
@aslakknutsen all I see here https://github.com/fabric8io/fabric8-init-tenant/blob/master/controller/tenant.go#L195-L243 is log statements and one call https://github.com/fabric8io/fabric8-init-tenant/blob/master/controller/tenant.go#L218 to service.UpdateNamespace() which I don't need. What am I missing? As I said without using any of this code the CLI is working fine for me (I don't want/need any DB updates)